frontend/src/component/project/Project/CreateProject/CreateProject.tsx

```diff
@@ -92,8 +94,10 @@ const CreateProject = () => {
                 projectId={projectId}
                 setProjectId={setProjectId}
                 projectName={projectName}
+                projectMode={projectMode}
```

Comment:
this is props drilling antipattern but I don't want to refactor as part of this PR